### PR TITLE
Dynamic attributes

### PR DESCRIPTION
**Pull request by wnolfm**

### CODE REVIEW

**Review by ryanwhitman — CHANGES_REQUESTED** [review]
Nolan, please be sure to run the fixer on this - https://github.com/oyova/wp-support/blob/feature/styles/CONTRIBUTING.md. Also, Rich can do the actual code review, but I assume these changes won't break any existing projects?

**Comment by wnolfm** [comment]
@ryanwhitman No projects are using this style of block opener yet. So no, it won't break anything.

**Comment by wnolfm** [comment]
Working on the class attribute. will reopen when complete.

File: src/Acf/Blocks/Block.php [hunk 1]
```diff
@@ -35,7 +37,7 @@ public function get_element_close(): string {
 	}
 
 	public function get_element_class(): string {
```
Comment (arjay-stanley) [inline src/Acf/Blocks/Block.php]: rename to get_element_classes

File: src/Acf/Blocks/Block.php [hunk 2]
```diff
@@ -63,47 +65,61 @@ public function get_element_type() {
...
+		if ( isset( $attributes['style'] ) ) {
+			$styles[] = $attributes['style'];
 		}
 
 		return $styles;
 	}
 
-	public function get_padding_styles(): array {
-		$styles = array();
+	public function get_wp_class(): array {
```
Comment (arjay-stanley) [inline src/Acf/Blocks/Block.php]: rename to get_wp_classes

**Review by arjay-stanley — CHANGES_REQUESTED** [review]
rename method `get_element_class` to `get_element_classes`
rename method `get_wp_class` to `get_wp_classes`

**Pull request merged**